bin/benchmark_pydy_code_gen.py

```diff
@@ -45,11 +40,8 @@ def run_benchmark(max_num_links, num_time_steps=1000):
         for i in range(n):
             parameter_vals += [arm_length, bob_mass]
 
-        # odeint arguments
-        x0 = hstack((0, pi / 2 * ones(len(results[3]) - 1), 1e-3 *
-                    ones(len(results[4]))))
-        args = {'constants': dict(zip(results[2], array(parameter_vals)))}
```

Comment:
I just noticed this. Sorry.

`x0` needs to be set in system as `sys.initial_conditions` and `constants` needs to be set as `sys.constants`.